source/jormungandr/jormungandr/interfaces/v1/GraphicalIsochron.py

```diff
@@ -0,0 +1,137 @@
+}
+
+
+class GraphicalIsochron(ResourceUri, ResourceUtc):
+
+    def __init__(self):
+        ResourceUri.__init__(self, authentication=False)
+        ResourceUtc.__init__(self)
+
+        modes = ["walking", "car", "bike", "bss"]
+        self.parsers = {}
+        self.parsers["get"] = reqparse.RequestParser(
+            argument_class=ArgumentDoc)
+        parser_get = self.parsers["get"]
+
+        parser_get.add_argument("from", type=unicode, dest="origin")
+        parser_get.add_argument("first_section_mode[]",
+                                type=option_value(modes),
+                                dest="origin_mode", action="append")
+        parser_get.add_argument("last_section_mode[]",
+                                type=option_value(modes),
+                                dest="destination_mode", action="append")
+        parser_get.add_argument("to", type=unicode, dest="destination")
+        parser_get.add_argument("datetime", type=date_time_format)
+        parser_get.add_argument("duration", type=int)
+        parser_get.add_argument("datetime_represents", dest="clockwise",
+                                type=dt_represents, default=True)
+        parser_get.add_argument("forbidden_uris[]", type=unicode, action="append")
+        parser_get.add_argument("max_transfers", type=int, default=42)
+        parser_get.add_argument("_current_datetime", type=date_time_format, default=datetime.utcnow(),
+                                description="The datetime used to consider the state of the pt object"
+                                            " Default is the current date and it is used for debug."
+                                            " Note: it will mainly change the disruptions that concern "
+                                            "the object The timezone should be specified in the format,"
+                                            " else we consider it as UTC")
+        parser_get.add_argument("max_walking_duration_to_pt", type=int,
+                                description="maximal duration of walking on public transport in second")
+        parser_get.add_argument("max_bike_duration_to_pt", type=int,
+                                description="maximal duration of bike on public transport in second")
+        parser_get.add_argument("max_bss_duration_to_pt", type=int,
+                                description="maximal duration of bss on public transport in second")
+        parser_get.add_argument("max_car_duration_to_pt", type=int,
+                                description="maximal duration of car on public transport in second")
+        parser_get.add_argument("walking_speed", type=float_gt_0)
+        parser_get.add_argument("bike_speed", type=float_gt_0)
+        parser_get.add_argument("bss_speed", type=float_gt_0)
+        parser_get.add_argument("car_speed", type=float_gt_0)
+        parser_get.add_argument("_min_car", type=int)
+        parser_get.add_argument("_min_bike", type=int)
```

Comment:
You can remove this two parameters, there are used in filters for journeys, but they won't be implemented in isochron